CHANGES

```diff
@@ -1,5 +1,6 @@
 clasp 3.3.4:
   * fixed: Atoms of incremental programs not always marked as frozen.
+  * added internal interface for user defined statistics
```

Comment:
For consistency, I’d write “user-defined“ with a hyphen here as well.

Reply:
Yep. Thanks.